### ISSUE TITLE
Wolvic launcher icon replaced by an Android generic one

### ISSUE THREAD

**Issue by javifernandez** [Igalia/wolvic#783]
## Configuration

Wolvic version: 1.4
Wolvic build ID:  96

Hardware:  Lynx R1

## Steps to Reproduce

1. Look at the Wolvic app in the launcher menu
2.
3.

## Current Behavior

A generic Android icon is shown

## Expected Behavior

The icon shipped in Wolvic's apk should be used

## Context

My guess is that the Wolvic's launcher icon is not defined correctly as Adaptative Icon [1]. In some systems the Legacy version is used instead, but apparently in Lynx this is not working properly; I'm speculating that it's due to some limitation / bug in the Lynx launcher. 

In any case, the Wolvic's launcher icons should be reviewed and, if possible, using the Adaptative Icon approach.

[1] https://developer.android.com/develop/ui/views/launch/icon_design_adaptive

## Error Logs and Stack Traces

<details open>

```
```

</details>

**Comment by svillar** [comment]
@javifernandez haven't we fixed this? If not what else should be done?

**Comment by javifernandez** [comment]
> @javifernandez haven't we fixed this? If not what else should be done?

No, but the issue is reproducible in Lynx R1 only, as far as I know. 
The workaround described [here](https://github.com/Igalia/wolvic/issues/783#issuecomment-1632433811) works, but I have not figured out the root cause of the problem. Given that it only happens in Lynx, it may be some platform-specific issue.
